### DISCUSSION TITLE
Non-Csla but development related questions

### DISCUSSION THREAD

**Discussion by Chicagoan2016** [MarimerLLC/csla#4152] [Ideas]
Fellow Csla developers,
Should we allow asking non-csla questions (within reason, of course)?
I value our fellow members' opinions way more than anyone else's.

Kind Regards

**Comment by kcabral817** [comment]
I do too.  This community is a lot more professional and helpful than SO and others who shall not be named.

Have a nice day.

Kevin




________________________________
From: Chicagoan ***@***.***>
Sent: Wednesday, August 7, 2024 1:51 PM
To: MarimerLLC/csla ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [MarimerLLC/csla] Non-Csla but development related questions (Discussion #4152)


Fellow Csla developers,
Should we allow asking non-csla questions (within reason, of course)?
I value our fellow members' opinions way more than anyone else's.

Kind Regards

**Comment by rockfordlhotka** [comment]
I'm fine with questions related to .NET app dev that are peripherally related to CSLA - like data access, deployment to various environments, etc.

It is a good learning opportunity for me, because I surely don't have deep knowledge in all these areas.

I don't want to see things go too far afield though - so I'd hope there's always some (even loose) connection with the sorts of things people need to address when building real-world apps using CSLA.
